apps/web/app/environments/[environmentId]/[actionsAndAttributesOption]/page.tsx

```diff
@@ -0,0 +1,21 @@
+import ActionClassesComponent from "@/app/environments/[environmentId]/[actionsAndAttributesOption]/(actions)/ActionClassesComponent";
+import ActionLoader from "@/app/environments/[environmentId]/[actionsAndAttributesOption]/(actions)/ActionLoader";
+import AttributeClassesComponent from "@/app/environments/[environmentId]/[actionsAndAttributesOption]/(attributes)/AttributeClassesComponent";
+import AttributeLoader from "@/app/environments/[environmentId]/[actionsAndAttributesOption]/(attributes)/AttributeLoader";
+import { Suspense } from "react";
+
+export default function ActiondAndAttributesPage({ params }) {
+  if (params.actionsAndAttributesOption === "actions")
+    return (
+      <Suspense fallback={<ActionLoader />}>
```

Comment:
If we do the folder structure like I suggested above I think we can also again switch to a `loading.tsx` here and have it more on par with next.js convention.